### DISCUSSION TITLE
Adjusting the speed and the pause time between paragraphs in output

### DISCUSSION THREAD

**Discussion by Masterof2D** [aedocw/epub2tts-edge#37] [Ideas]
I want to start by saying that this has been by far the best epub-to-audiobook tool, and I appreciate the work that has gone into this project so much! Before using this, I would use Microsoft Edge read aloud natively. When I used read-aloud natively, the pauses between paragraphs were just slightly shorter, and it sounded much more natural; if it were possible to add an option to adjust or shorten the pause between paragraphs, it would be very appreciated. It also would be nice to be able to set the speed of the output, it is less important since most audiobook readers support speed control with no problem, but it would be nice to have it since the default output is very slowly paced

**Comment by aedocw** [comment]
Thanks, glad you're enjoying it!

Making the delay between paragraphs adjustable would be relatively easy. The default is 2000 milliseconds (2 seconds), but should be nothing at all to turn that into a command line option.

**Comment by aedocw** [comment]
Go ahead and try adding `--pause 0` (or any number of milliseconds) to set the pause time after each paragraph. Let me know what you think, it seems to work well.

I think it still gets the same 1.2 second pause at the end of each sentence, which now that I type this makes me think I should re-do this and have `--sentencepause` and `--paragraphpause` ... Maybe I'll do that real quick :)
